rust/processor/src/models/fungible_asset_models/v2_fungible_asset_activities.rs

```diff
@@ -182,6 +187,7 @@ impl FungibleAssetActivity {
         transaction_version: i64,
         transaction_timestamp: chrono::NaiveDateTime,
         block_height: i64,
+        fee_statement: Option<FeeStatement>,
```

Comment:
Does the comment mean that we only have storage refund for fungible asset v1 (not v2)?

Reply:
I added this to coin_activities as well.

Reply:
I mean it seems like this function should only be indexing v1 activities, but I guess it's not anymore? So let's change the variables naming to clarify.

Reply:
sorry i don't understand your question. this is used for gas. not sure what function you're referring to.

Reply:
I'm referring to `get_gas_event`

Reply:
ah i see. regardless of the naming, the plan is to deprecate coin_activities at some point down the line. Idk if we need to beat down on this variable naming. I do want to migrate all this code over to fungible_asset_activities instead of doing this.